src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onPicking: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onPicking,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn(
+                { 'is-active': tab.id === selectedTabId },
+              )}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (selectedTab.id !== tab.id) {
+                    onPicking(tab);
+                  }
+                }}
```

Comment:
move function to variable. If you will be confused with `tab.id` use data attributes [Usage in React part](https://www.pluralsight.com/guides/how-to-access-custom-attributes-from-aevent-object-in-react)